src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    status: 'Under construction',
+    show: false,
+  },
+  reducers: {
+    statusCategories: (state) => (state.status),
+    toggleShow: (state) => (
+      {
+        ...state,
+        show: !state.show,
+      }
+    ),
+  },
+});
+
+export const { statusCategories, toggleShow } = categoriesSlice.actions;
+export default categoriesSlice.reducer;
```

Comment:
- [ ] Kindly use the action creators and reducer in the `categories.js` instead of this one as a slice file. 👍 

Kindly see the requirement below

![image](https://user-images.githubusercontent.com/77106962/160717972-772380b5-0f0b-4351-bd05-b57b976ba7dd.png)